sydent/db/threepid_associations.py

```diff
@@ -95,6 +95,41 @@ def getMxid(self, medium, address):
 
         return row[0]
 
+    def getMxids(self, threepid_tuples):
+        cur = self.sydent.db.cursor()
+
+        cur.execute("CREATE TEMPORARY TABLE tmp_getmxids (medium VARCHAR(16), address VARCHAR(256))");
```

Comment:
sqlite's docs are a bit vague, but yes, they do seems to be (and we only ever have one connection at a time anyway).